README.md

```diff
@@ -83,6 +83,12 @@ OAUTH_PROVIDERS = [
 CUSTOM_SECURITY_MANAGER = CustomSecurityManager
 ```
 
+As an alternative, you can also simply add a redirect variable to the url so as to redirect after logging in. This can be added as below
+
+```
+'https://example.com/login/provider?redirect=/superset/dashboard/3/'
```

Comment:
Why does the word `provider` need to be here?

Reply:
It's just an example of a url with a provider. It could also be without